### ISSUE TITLE
Supplies consumed when Kerbals thawed (USI-LS)

### ISSUE THREAD

**Issue by MrKiel** [JPLRepo/DeepFreeze#40]
Same issue as described for TAC-LS. 

Using USI-LS version 0.3.12.0 and deep freeze 0.20.3, when Kerbals are thawed, supplies are consumed from the entire time they were frozen. 

For example, if one Kerbal is frozen for 100 days, and then thawed, 100 days worth of supplies are consumed immediately.

**Comment by JPLRepo** [comment]
This is not an actual issue with DeepFreeze but an issue with USI LS 0.3.12.0.
I can reproduce this behaviour with a standard vessel (no DeepFreeze) and USI LS installed. It occurs whenever a vessel is left un-attended for any period of time.
Fix Pull request sent to roverdude.
https://github.com/BobPalmer/USI-LS/pull/85

**Comment by Gnurfos** [comment]
Hi, I think this issue is back (I'm experiencing the symptoms). Would you be so kind as to check ? I know the fix as in USI-LS last time, but currently it used to refer to a global-ish part.protoModuleCrew.Count but [now](https://github.com/BobPalmer/USI-LS/blob/master/Source/USILifeSupport/ModuleLifeSupportSystem.cs) it seems to have its own attribute instead (_currentCrew). I have no idea if this is easily fixable and on which side.

**Comment by JPLRepo** [comment]
I tried to reproduce this myself last night and could not.
I ran a test. There appears to be nothing wrong with DeepFreeze and if there is an issue it would be with USI-LS.

**Comment by Gnurfos** [comment]
I noticed that this only happens when thawing the first kerbal (I had 2 frozen, no one else on board). Using hyperedit to keep supplies topped-off during the unwanted "adjustment", I managed to prevent the first K thawed from going tourist, and when then thawing the 2nd one, I did nothing, yet suplies did not drain.
So it seems more triggered by the vessel changing from 0 active Kerbal to 1, rather than upon individual K thawing. Could it be that in your test there were other non-frozen Kerbals on board ?

**Comment by JPLRepo** [comment]
Yes indeed. I just ran the same test. It seems to only occur when you freeze all kerbals on-board and the resulting vessel with no kerbals on board is run for some time and then you thaw a kerbal.
It is not a DeepFreeze bug but a USI LS bug so I'd suggest you raise a bug report on the USI LS github repro.
In fact what I saw was LS supplies going down even though Jeb was frozen and there was no one on board.
When I thawed him he gobbled up a bunch more LS supplies but then they suddenly re-appeared again.

![2017-01-09 11_16_30-kerbal space program](https://cloud.githubusercontent.com/assets/9981382/21754878/31c88942-d65d-11e6-873d-a2b653d40e7d.png)
![2017-01-09 11_16_53-kerbal space program](https://cloud.githubusercontent.com/assets/9981382/21754879/3483acc0-d65d-11e6-9726-5e0c39b5b796.png)
